manual/installation.tex

```diff
@@ -459,8 +457,8 @@ \subsection{Build scripts}
 
 cmake                                                \
   -D CMAKE_BUILD_TYPE=Debug                         \
-  -D Libxml2_INCLUDE_DIRS=/usr/include/libxml2      \
-  -D Libxml2_LIBRARY_DIRS=/usr/lib/x86_64-linux-gnu \
+  -D LIBXML2_INCLUDE_DIR=/usr/include/libxml2      \
+  -D LIBXML2_LIBRARY=/usr/lib/x86_64-linux-gnu \
```

Comment:
I checked that LIBXML2_LIBRARY is not a path but the exact library.
For example.
-D LIBXML2_LIBRARY=/usr/lib64/libxml2.so

The above
`Library location for libxml2` probably is not clear enough.

Reply:
Fixed